streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -179,6 +179,7 @@ public void testStateGlobalThreadClose() throws Exception {
         builder.globalTable("anyTopic");
         props.put(StreamsConfig.NUM_STREAM_THREADS_CONFIG, numThreads);
         final KafkaStreams streams = new KafkaStreams(builder.build(), props);
+        streams.setGlobalStateRestoreListener(new MockStateRestoreListener());
```

Comment:
Do we need this? The test doesn't actually use or test it so it seems irrelevant

Reply:
Nope, good catch